core/includes/update.inc

```diff
@@ -89,10 +89,17 @@ function update_prepare_bootstrap() {
   $active_config = config_get_config_storage('active');
   $staging_config = config_get_config_storage('staging');
 
-  if (!$settings_exist || !$active_config->isInitialized() || !$staging_config->isInitialized()) {
+  if (!$settings_exist || !$active_config->isInitialized()) {
     backdrop_install_config_directories();
   }
 
+  // Check for a staging directory
```

Comment:
```suggestion
  // Warn if staging directory is not found.
```

Reply:
...alternative for consideration:

```suggestion
  // Warn if staging directory is not found.
```

Not sure if we want to add a "but do not prevent updates" there.